envoy/http/persistent_quic_info.h

```diff
@@ -9,5 +11,7 @@ struct PersistentQuicInfo {
   virtual ~PersistentQuicInfo() = default;
```

Comment:
I think with the move to Envoy, we should talk more about the lifetime and expecations.
Also a comment on why the actual info isn't here, to limit scope of quic variables.

Reply:
done